oe_corporate_blocks.post_update.php

```diff
@@ -10,9 +10,33 @@
 /**
  * Delete the config for Site Switcher.
  */
-function oe_corporate_blocks_post_update_20001() {
+function oe_corporate_blocks_post_update_20001(&$sandbox): void {
   // Delete the config.
   \Drupal::configFactory()
     ->getEditable('oe_corporate_blocks.data.site_switcher')
     ->delete();
 }
+
+/**
+ * Only display "European Commission website" link in footer.
+ */
+function oe_corporate_blocks_post_update_20002(&$sandbox): void {
```

Comment:
This needs to be called the same as on 1.x branch, or else it will be ran twice:
```
oe_corporate_blocks_post_update_10001
```

Reply:
pushed